README.md

```diff
@@ -154,6 +155,8 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Open Policy
 - [Go SDK](https://github.com/open-policy-agent/opa/tree/main/internal/wasm/sdk) - a small Go library for using WebAssembly compiled Open Policy Agent Rego policies
 - [JVM](https://github.com/sangkeon/java-opa-wasm) - Java SDK for calling Wasm-compiled policies. Uses wasmtime.
 - [Rust](https://github.com/matrix-org/rust-opa-wasm) - A crate to use OPA policies compiled to Wasm
+- [OpaDotNet](https://github.com/me-viper/OpaDotNet) - Open Policy Agent (OPA) WebAssembly dotnet core SDK
+- [OpaDotNet.Compilation](https://github.com/me-viper/OpaDotNet.Compilation) - dotnet core backend for packaging Open Policy Agent Rego policies and data files into WASM policy bundles
```

Comment:
Why not put it near [this line](https://github.com/StyraInc/awesome-opa/pull/58/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R153)? Since it's both .NET, I think that would make sense. 🤔

Reply:
Sure, no problem. Wasn't sure where it belongs.